tests/ValueParsers/TimeParserTest.php

```diff
@@ -21,7 +22,7 @@ class TimeParserTest extends ValueParserTestBase {
 	 * @deprecated since 0.3, just use getInstance.
 	 */
 	protected function getParserClass() {
-		return 'ValueParsers\TimeParser';
+		throw new \LogicException( 'Should not be called, use getInstance' );
```

Comment:
The relevant change was done in #44 and is not part of this PR. When `getInstance` is implemented `getParserClass` is never called.